Blackjack.Tests/DeckShould.cs

```diff
@@ -0,0 +1,103 @@
+            {
+                new Card(CardRank.Ace, CardSuit.Clubs),
+                new Card(CardRank.Ace, CardSuit.Diamonds),
+                new Card(CardRank.Ace, CardSuit.Hearts),
+                new Card(CardRank.Ace, CardSuit.Spades)
+            };
+
+            var result = BlackjackHelper.ListsOfCardsAreEqual(cards1, cards2);
+
+            Assert.True(result);
+        }
+
+        [Fact]
+        public void NotBeEquivalent_GivenTwoDifferentLists_ofCards()
+        {
+            var cards1 = new List<Card>()
+            {
+                new Card(CardRank.Two, CardSuit.Clubs),
+                new Card(CardRank.Ace, CardSuit.Diamonds),
+                new Card(CardRank.Ace, CardSuit.Hearts),
+                new Card(CardRank.Ace, CardSuit.Spades)
+            };
+            var cards2 = new List<Card>()
+            {
+                new Card(CardRank.Ace, CardSuit.Clubs),
+                new Card(CardRank.Ace, CardSuit.Diamonds),
+                new Card(CardRank.Ace, CardSuit.Hearts),
+                new Card(CardRank.Ace, CardSuit.Spades)
+            };
+
+            var result = BlackjackHelper.ListsOfCardsAreEqual(cards1, cards2);
+
+            Assert.False(result);
+        }
+
+        [Fact]
+        public void BeEquivalent_GivenTwoDecksOfCards()
+        {
+            var cards1 = new List<Card>()
+            {
+                new Card(CardRank.Ace, CardSuit.Clubs),
+                new Card(CardRank.Ace, CardSuit.Diamonds),
+                new Card(CardRank.Ace, CardSuit.Hearts),
+                new Card(CardRank.Ace, CardSuit.Spades)
+            };
+            var cards2 = new List<Card>()
+            {
+                new Card(CardRank.Ace, CardSuit.Clubs),
+                new Card(CardRank.Ace, CardSuit.Diamonds),
+                new Card(CardRank.Ace, CardSuit.Hearts),
+                new Card(CardRank.Ace, CardSuit.Spades)
+            };
+            var deck1 = new Deck(cards1);
+            var deck2 = new Deck(cards2);
+
+            var result = BlackjackHelper.DecksOfCardsAreEqual(deck1, deck2);
+
+            Assert.True(result);
+        }
+
+        [Fact]
+        public void NotBeEquivalent_GivenTwoDifferentDecks()
+        {
+            var cards1 = new List<Card>()
+            {
+                new Card(CardRank.Two, CardSuit.Clubs),
+                new Card(CardRank.Ace, CardSuit.Diamonds),
+                new Card(CardRank.Ace, CardSuit.Hearts),
+                new Card(CardRank.Ace, CardSuit.Spades)
+            };
+            var cards2 = new List<Card>()
+            {
+                new Card(CardRank.Ace, CardSuit.Clubs),
+                new Card(CardRank.Ace, CardSuit.Diamonds),
+                new Card(CardRank.Ace, CardSuit.Hearts),
+                new Card(CardRank.Ace, CardSuit.Spades)
+            };
+
+            var result = BlackjackHelper.ListsOfCardsAreEqual(cards1, cards2);
+
+            Assert.False(result);

```

Comment:
Why are we needing to test for equivalence? Remember that tests are for behaviours or features, not implementation. Are we intending on having different decks being compared as a behaviour of the Blackjack game?